### PR TITLE
UserAgent Integration in Headers

### PR DESCRIPTION
**Pull request by twdnhfr**
Maybe get this from a config?

### CODE REVIEW

**Comment by alaingilbert** [comment]
Thanks for this contribution :)

Can you explain what `planet_get_next_best_mine` is doing ?

File: ogame/__init__.py [hunk 1]
```diff
@@ -26,7 +26,8 @@ def login(self):
         payload = {'kid': '',
                    'uni': self.server_url,
                    'login': self.username,
-                   'pass': self.password}
+                   'pass': self.password,
+                   'User-Agent': 'Mozilla/5.0 (Windows NT 10.0; WOW64; rv:48.0) Gecko/20100101 Firefox/48.0'}
```
Comment (alaingilbert) [inline ogame/__init__.py:30]: This is not where the user-agent should be.
It should be put in the headers, not payload.

**Comment by twdnhfr** [comment]
ah ok i thought you just named the headers that way. 

planet_get_next_best_mine wasnt meant to be merged here :) i was basically testing Pullrequests. But what it will do is deciding what the best mine to build on a planet, based on current levels and such.

I will close that PR :)

**Comment by alaingilbert** [comment]
https://github.com/alaingilbert/pyogame/blob/master/ogame/__init__.py#L15
This is how to change the user agent.
You can change it in your code with:

```
ogame.session.headers.update({'User-Agent': 'NEW_VALUE})
```

Or we could make a new function `set_user_agent(user_agent)`

**Pull request closed without merging**